src/main/java/mate/jdbc/model/Driver.java

```diff
@@ -6,13 +6,41 @@ public class Driver {
     private Long id;
     private String name;
     private String licenseNumber;
+    private Car car;
```

Comment:
The Driver class should not have a reference to a Car. A driver can have multiple cars, and a car can have multiple drivers. Consider creating a separate relationship class or use a Many-to-Many relationship in a database.